src/wmtk/simplex/link_condition.cpp

```diff
@@ -5,16 +5,18 @@
 namespace wmtk::simplex {
 bool link_condition_closed_trimesh(const TriMesh& mesh, const Tuple& edge)
 {
-    SimplexCollection lnk_a = link(mesh, Simplex(PrimitiveType::Vertex, edge), true); // link(a)
-    SimplexCollection lnk_b = link(
-        mesh,
-        Simplex(PrimitiveType::Vertex, mesh.switch_tuple(edge, PrimitiveType::Vertex)),
-        true); // link(b)
-    SimplexCollection lnk_ab = link(mesh, Simplex(PrimitiveType::Edge, edge), true); // link(ab)
+    const Simplex v_a = Simplex(PrimitiveType::Vertex, edge);
```

Comment:
You can use `Simplex::vertex(edge)` and `Simplex::edge(edge)`. That's a bit shorter.